partials/instagram.php

```diff
@@ -1,12 +1,39 @@
-<?php 
+<section id="instagram-feed" class="row"> 
+
+<?php
 $instagram_handle = IGV_get_option( '_igv_instagram_handle' ); 
-?>
-<section class="row"> 
+if ($instagram_handle) {
```

Comment:
you call IGV get option twice. Why not just call it in the `get_instagram_feed()` function and check is set up there?